src/main/kotlin/mate/academy/model/Figure.kt

```diff
@@ -0,0 +1,5 @@
+package mate.academy.model
+
+sealed class Figure(val color: Color): AreaCalculator, Drawable {
```

Comment:
As possible solution Fugure can be abstract class with getArea() and draw() methods.
not mistake just FYI